projects/chef-infra.md

```diff
@@ -6,43 +6,50 @@ The Chef Infra project includes chef-client and all directly related projects.
 
 **Triage Schedule**: Mondays at 10:00am PST
 
-### Project Members
+## Project Members
 
-#### Project Owners
+### Project Owners
 
 Bryan McLellan
-  - Github: [btm](https://github.com/btm)
-  - Slack: @btm
+
+- Github: [btm](https://github.com/btm)
+- Slack: @btm
 
 Lamont Granquist
-  - Github: [lamont-granquist](https://github.com/lamont-granquist)
-  - Slack: @lamont
+
+- Github: [lamont-granquist](https://github.com/lamont-granquist)
+- Slack: @lamont
 
 Phil Dibowitz
-  - Github: [jaymzh](http://github.com/jaymzh)
-  - Slack: @phil
+
+- Github: [jaymzh](http://github.com/jaymzh)
+- Slack: @phil
```

Comment:
huh. That's actually not true, I'm @jaymzh on slack. If you don't wanna fix this in this PR I'll fix it in a separate one.